iDiff/utils/image_utils.go

```diff
@@ -72,3 +168,18 @@ func copyToFile(outfile string, r io.Reader) error {
 
 	return nil
 }
+
+func checkImageID(image string) bool {
```

Comment:
This function may be more useful farther up the stack to help validate arguments.  Maybe in differs.go as we also expand to tar arguments. Also I think we had talked about trying to isolate Docker dependent things in this file.